pipeline_steps/common.groovy

```diff
@@ -904,4 +906,20 @@ void setTriggerVars(){
   print ("Trigger: ${env.RE_JOB_TRIGGER} (${env.RE_JOB_TRIGGER_DETAIL})")
 }
 
+// add wrappers that should be used for all jobs.
+// max log size is in MB
+void globalWraps(Closure body){
+  // global timeout is long, so individual jobs can set shorter timeouts and
+  // and still have to cleanup, archive atefacts etc.
+  timeout(time: 10, unit: 'HOURS'){
+    wrap([$class: 'LogfilesizecheckerWrapper', 'maxLogSize': 200, 'failBuild': true, 'setOwn': true]) {
```

Comment:
What does `'setOwn'` do?

Reply:
AFAICT it makes maxLogSize active, otherwise a value from Jenkins global config is used. I'd prefer to keep stuff out of Jenkins global config as its not in SCM.